astro/src/content/docs/apis/_user-request-body.mdx

```diff
@@ -132,10 +132,8 @@ You must specify either the **email** or the **username** or both for the User.
   <APIField name="user.mobilePhone" type="String" optional>
```

Comment:
Did we decide to deprecate this field? Should it be noted here?

Reply:
I don't know that we did decide to deprecate it.